### PR TITLE
changes for approval workflow

### PR DESCRIPTION
**Pull request by Chocrates**

### CODE REVIEW

**Comment by zkoppert** [comment]
@Chocrates It looks like the test suite fails. It doesn't seem to like the received value.

```
expect(received).toHaveBeenCalledTimes(expected)

    Matcher error: received value must be a mock or spy function

    Received has value: undefined

      21 |   await run();
      22 |
    > 23 |   expect(octomock.mockFunctions.getInput).toHaveBeenCalledTimes(6);
         |                                           ^
      24 |   expect(octomock.mockFunctions.createComment).toHaveBeenCalled();
      25 | });
      26 |

      at __tests__/main.test.ts:23:43
      at fulfilled (__tests__/main.test.ts:5:58)
```

**Comment by zkoppert** [comment]
Any idea why the status checks are perpetually "waiting for status to be reported"?

File: __tests__/main.test.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ beforeAll(() => {
 });
 test('Main', async () => {
   await run();
-  expect(octomock.mockFunctions.getInput).toHaveBeenCalledTimes(3);
+
+  expect(octomock.mockFunctions.getInput).toHaveBeenCalledTimes(6);
   expect(octomock.mockFunctions.createComment).toHaveBeenCalled();
```
Comment (zkoppert) [inline __tests__/main.test.ts]: test fails here

Comment (Chocrates) [reply __tests__/main.test.ts]: I just packaged octomock into an npm package and added some breaking changes with a refactor.  I bet that is part of the issue.  Let me try to pin this to `v0.0.1`

**Pull request merged**